.typo-ci.yml

```diff
@@ -0,0 +1,44 @@
+---
+dictionaries:
+  - de
+  - en
+  - en_GB
+  - en_CA
+  - es
+  - fr
+  - it
+
+excluded_files:
+  - *.yml
```

Comment:
@vbrazo I _think_ this should handle `.typo-ci.yml`, right?

Reply:
yes 👍